Makefile

```diff
@@ -30,14 +39,46 @@ e2e: deploy
 $(TARGETS): vendor/modules.txt
 	./scripts/$@
 
-build-subctl: pkg/subctl/operator/common/embeddedyamls/yamls.go $(shell find pkg/subctl/ -name "*.go")
+clean:
+	rm -f $(BINARIES) $(CROSS_BINARIES) $(CROSS_TARBALLS)
+
+build: operator-image binaries
+
+build-cross: $(CROSS_TARBALLS)
+
+binaries: $(BINARIES)
```

Comment:
I'd skip this and just have `build` rely on `bin/subctl` unless you think someone would use `make binaries` :)

Reply:
Indeed, done!